### PR TITLE
Fix a few typos

### LINKED ISSUE #249: Fix typo in providers API doc
Need to fix typo "postional" -> "positional".

### PR DESCRIPTION
**Pull request by kinow**
Closes #249

@rmk135 found a few more typos beside the one from #249. 

Cheers
Bruno

### CODE REVIEW

**Comment by kinow** [comment]
Added a second commit with more changes, now to the documentation. Renamed one example class too. Happy to undo changes, or include new ones to the PR if necessary. Thanks for `python-dependency-injector`!

**Comment by rmk135** [comment]
That's cool. Thank you for the help. Please, take a look at the review comments.

File: docs/providers/factory.rst [hunk 1]
```diff
@@ -160,7 +160,7 @@ aggregates other :py:class:`Factory` providers.
 
     :py:class:`FactoryAggregate` is not overridable. Calling of 
     :py:meth:`FactoryAggregate.override` will result in raising of an 
-    expection.
+    expectation.
```
[the remark this answers is not in the document]
Comment (kinow) [reply docs/providers/factory.rst]: Done!

File: docs/providers/overriding.rst [hunk 2]
```diff
@@ -9,7 +9,7 @@ This gives opportunity to make system behaviour more flexible at some point.
 The main feature is that while your code is using providers, it depends on 
 providers, but not on the objects that providers provide. As a result of this, 
 you can change providing by provider object to a different one, but still
-compatible one, without chaning your previously written code.
+compatible one, without chaining your previously written code.
```
[the remark this answers is not in the document]
Comment (kinow) [reply docs/providers/overriding.rst]: Done!

**Comment by kinow** [comment]
Edited the commit to fix the docs, and now pointing to `develop` branch. Thanks for the review @rmk135 !

**Comment by rmk135** [comment]
@kinow can you leave any comment at #249 . I want to assign it to you before closing. That's github "feature" - no way to assign an issue to somebody who is not involved at least with a comment :)

**Comment by kinow** [comment]
Done @rmk135 , and thanks for telling me about this feature. I thought only members of a team in an org/repo could be assignees. Good to know!

**Comment by rmk135** [comment]
Yep, I also didn't know this before. Googled and found this one - https://stackoverflow.com/questions/41171882/how-to-assign-an-issue-to-non-contributors

I've added you to the list list of contributors that is distributed with every copy - https://github.com/ets-labs/python-dependency-injector/blob/develop/CONTRIBUTORS.rst

Also I've made a not in changelog - https://github.com/ets-labs/python-dependency-injector/blob/develop/docs/main/changelog.rst#development-version

Changes will appear on PyPi and ReadTheDocs in a couple of days.

Sincere Thanks,
Roman

**Pull request merged**